lua/autorun/netstream.lua

```diff
@@ -4,7 +4,7 @@ AddCSLuaFile()
 
 net.Stream = {}
 net.Stream.SendSize = 20000 --This is the size of each packet to send
-net.Stream.Timeout = 10 --How long to wait for client response before cleaning up
+net.Stream.Timeout = 30 --How long to wait for client response before cleaning up
```

Comment:
Will need to open PRs for other addons. The Initialize hook was just for debug testing